vkcs/resource_vkcs_db_config_group.go

```diff
@@ -0,0 +1,215 @@
+package vkcs
+
+import (
+	"context"
+	"log"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func resourceDatabaseConfigGroup() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceDatabaseConfigGroupCreate,
+		ReadContext:   resourceDatabaseConfigGroupRead,
+		UpdateContext: resourceDatabaseConfigGroupUpdate,
+		DeleteContext: resourceDatabaseConfigGroupDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"datastore": {
+				Type:     schema.TypeList,
+				Required: true,
+				ForceNew: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"version": {
+							Type:     schema.TypeString,
+							Required: true,
+							ForceNew: true,
+						},
+						"type": {
+							Type:     schema.TypeString,
+							Required: true,
+							ForceNew: true,
+						},
+					},
+				},
+			},
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"values": {
+				Type:     schema.TypeMap,
+				Required: true,
+				ForceNew: false,
+				Elem:     &schema.Schema{Type: schema.TypeString},
+			},
+			"updated": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"created": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"datastore_version_id": {
```

Comment:
do we really need it? do we really need it separately of datastore section?

Reply:
Removed